src/style.css

```diff
@@ -40,7 +53,8 @@ h1 {
 }
 
 .single-score:nth-child(even) {
-  background-color: rgb(226, 224, 224);
+  /* background-color: rgb(226, 224, 224); */
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

Reply:
Resolved. ✔️